OCAT/client/pages/Assessments/NewAssessment.jsx

```diff
@@ -1,16 +1,227 @@
+
 import React from 'react';
 import { Button, Form } from 'react-bootstrap';
+import { useForm } from 'react-hook-form';
 import { AssessmentService } from '../../services/AssessmentService';
 
 export const NewAssessment = () => {
+  const { formState: { errors }, handleSubmit, register } = useForm({
+    defaultValues: {
+      altercations_with_cats: ``,
+      altercations_with_owner: ``,
+      cat_name: ` `,
+      cat_date_of_birth: ``,
+      hisses_at_strangers: ``,
+      plays_with_dogs: ``,
+      previous_contact: ``,
+    },
+
+  });
 
   // create a form that utilizes the "onSubmit" function to send data to OCAT/client/services/AssessmentService.js and
   // then onto the OCAT/server/routes/AssessmentAPI express API
+
   const onSubmit = async (data) => {
-    await AssessmentService.submit(data);
+    try
+    {
+      const num1 = parseInt(data.previous_contact);
+      const num2 = parseInt(data.altercations_with_cats);
+      const num3 = parseInt(data.altercations_with_owner);
+      const num4 = parseInt(data.plays_with_dogs);
+      const num5 = parseInt(data.hisses_at_strangers);
+      let risk_level = ``;
+      const created_at = new Date();
+
+      const score = num1 + num2 + num3 + num4 + num5;
+      console.log(num1);
+      console.log(num2);
+      console.log(num3);
+      console.log(num4);
+      console.log(num5);
+      console.log(`score:${score}`);
+
+      switch (score) {
+        case 2:
+        case 3:
+          risk_level = `Medium Risk`;
+          break;
+        case 4:
+        case 5:
+          risk_level = `High Risk`;
+          break;
+        default:
+          risk_level = `low Risk`;
+      }
+      console.log(risk_level);
+      console.log(created_at);
+      console.log(`data`, data);
+
+      await AssessmentService.submit({ score, risk_level, created_at, data });
+    }
+    catch (error) {
+      throw new Error(`${error.response.statusText} - ${error.response.data.message}`);
+    }
```

Comment:
```suggestion
    try
    {
      const response1 = parseInt(data.previous_contact);
      const response2 = parseInt(data.altercations_with_cats);
      const response3 = parseInt(data.altercations_with_owner);
      const response4 = parseInt(data.plays_with_dogs);
      const response5 = parseInt(data.hisses_at_strangers);
      const score = response1 + response2 + response3 + response4 + response5;
     
      const created_at = new Date();

      let risk_level = ``;
      switch (score) {
        case 1:
          risk_level = `Low`;
          break;
        case 2:
        case 3:
          risk_level = `Medium`;
          break;
        case 4:
        case 5:
          risk_level = `High`;
          break;
        default:
          throw new Error(`unable to map risk level`);
      }

      await AssessmentService.submit({ score, risk_level, created_at, data });
    }
    catch (error) {
      throw new Error(`${error.response.statusText} - ${error.response.data.message}`);
    }
```

This is pretty good, but moving the variables closer to the code relevant to them will make it easier to read in the future (help your other devs out!).

Using more accurate variable names is better, too.  `num1` just tells us it's a random number with no context.  `response1` gives more info because we know it's a response directly from the form

Also, it's pretty lazy to leave the risk_level options inconsistent like this.  They should follow the same format.  In addition, including `risk` in the value is redundant.  We already know it's a risk level.

Finally, using the default case in this was is a little sloppy.  It's more clear and readable if you explicitly spell out all the possible cases.  The default is better having an error message (because it's an unexpected possibility).  That way, if you see the error there's definitely a bug you can fix.